Templates/GitlabCIPipeline/README.md

```diff
@@ -38,7 +39,8 @@ The pipeline uses the Gitlab concepts: `Stage`and `Jobs`.
 
 To leverages this template, access to a Gitlab CI/CD environment is required, and an Gitlab runner must be configured to connect to your mainframe environment. Please review the setup instructions of this [document](https://www.ibm.com/support/pages/system/files/inline-files/Integrating%20IBM%20zOS%20platform%20in%20CICD%20pipelines%20with%20GitLab%20-%20v1.7_1.pdf).
 
-The template leverages the [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli/) to issue command invoking the [Common Backend scripts](../Common-Backend-Scripts/) on USS.
+The template leverages [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli/) to issue command invoking the [Common Backend scripts](../Common-Backend-Scripts/) on USS. It specifically uses the `issue unix-shell` command which allows streaming of the console outputs back the Zowe CLI task.
```

Comment:
On the last sentence, can you rephrase to something like:
It specifically uses the `issue unix-shell` command which allows streaming of the console outputs back to the Zowe Command-Line Interface.